lib/si-data-pg/src/lib.rs

```diff
@@ -174,6 +175,10 @@ pub struct PgPoolConfig {
     pub pool_timeout_wait_secs: Option<u64>,
     pub pool_timeout_create_secs: Option<u64>,
     pub pool_timeout_recycle_secs: Option<u64>,
+    pub pool_total_connection_lifetime_secs: u64,
+    pub pool_idle_connection_lifetime_secs: u64,
+    /// If set to `None`, the eviction task won't be started
+    pub pool_lifetime_check_interval_secs: Option<u64>,
```

Comment:
The expectation here is that the AWS parameter store will inject this values without any other code changes, but I may be wrong